README.md

```diff
@@ -1,2 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Leaderboard
-Set up a JavaScript project for the Leaderboard list app
+
+- In this project, I built a simple Leaderboard page. Built with HTML, CSS.
+
+## Built With
+
+- HTML
+- CSS
+- Es6
+- Gitflow
+- Visual Studio Code
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] to make your repo professional kindly update the Getting Started, Install & Setup sections in your readme file, and try to guide the user on how to install & setup your project like clone, run npm install, etc ...